apps/ui/components/Carousel.tsx

```diff
@@ -13,6 +13,7 @@ type Props = {
   cellSpacing?: number;
   wrapAround?: boolean;
   hideCenterControls?: boolean;
+  slideIndex?: number;
```

Comment:
It's also in the wrong branch :D This is used in the ui/ux-improvements branch to go to the correct "page"(== slide) of quick reservation slots, when a time is selected which isn't on the first page (otherwise the "quick slot selector" would show the first page, and thus "no selected slot" - even tho there would actually be a slot selected, but it's e.g. on the third page and hidden because of that).

Reply:
Actually I take back my words about it being in the wrong branch. It's in use even here - in the eventuality that the query-params specify a time slot which is not on the first page/slide of the available quick reservation slots for the day - so the described feature it is warranted even here.